ethereum/contracts/SwapCreator.sol

```diff
@@ -18,17 +18,18 @@ contract SwapCreator is ERC2771Context, Secp256k1 {
     }
 
     struct Swap {
-        // individual swap creator, Alice
+        // the swap initiator, Alice
+        // Alice is the only one who can call `refund`
         address payable owner;
-        // address allowed to claim the ether in this contract
+        // address allowed to claim the ether for this swap, Bob
```

Comment:
Saying that Bob is an address would sound okay if Alice was described the same way, but we said she is "the swap initiator".  Here's a starter idea on how synchronize description styles:
```
the swap offerer (maker?), who can claim the ether of this swap, Bob
```

Reply:
I updated the comment for Alice to match the one for Bob, I think that's clearer